### PR TITLE
665 disable removing items for tools

### PR DESCRIPTION
**Pull request by nimeshgarg**

### CODE REVIEW

File: source/core/src/main/com/csse3200/game/components/player/InventoryComponent.java [hunk 1]
```diff
@@ -61,13 +63,15 @@ public class InventoryComponent extends Component {
      */
     private int maxInventorySize = 30; // default size 30
 
+    private final List<String> forbiddenItems = Arrays.asList("shovel","hoe","watering_can","scythe","sword","gun");
```
Comment (JTrenerry) [inline source/core/src/main/com/csse3200/game/components/player/InventoryComponent.java:66]: fishing rod :(

**Review by JTrenerry — CHANGES_REQUESTED** [review]
Left comments

File: source/core/src/main/com/csse3200/game/components/player/InventoryComponent.java (hunk 1 above)
Comment (nimeshgarg) [reply source/core/src/main/com/csse3200/game/components/player/InventoryComponent.java:66]: Added fishing rod

**Comment by nimeshgarg** [comment]
> Left comments

Fixed

**Review by tristanduncombe — APPROVED** [review]
LGTM

**Review by JTrenerry — APPROVED** [review]

**Pull request merged**